lib/app/sequence_base.rb

```diff
@@ -725,6 +725,27 @@ def resolve_element_from_path(element, path)
         nil
       end
 
+      def get_value_for_search_param(element)
+        case element
+        when FHIR::Period
+          element.start || element.end
+        when FHIR::Reference
+          element.reference
+        when FHIR::CodeableConcept
+          resolve_element_from_path(element, 'coding.code')
+        when FHIR::Identifier
+          element.value
+        when FHIR::Coding
+          element.code
+        when FHIR::HumanName
+          element.family || element.given || element.text
```

Comment:
`given` is an array, so I think we'd need `given.first`.

Reply:
fixed